_posts/patterns/2015-04-15-icons.md

```diff
@@ -910,18 +932,18 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Hidden, Hide
+            eye-slash
```

Comment:
Re: accessibility, aren't these icons also used to "hide" and "show/reveal" things? Aren't they used as actions in addition to states? If so, maybe we can make notes that indicate to have accessible text that represents the purpose of the icon.